distribution/lib/Standard/Base/0.0.0-dev/src/Data/Statistics.enso

```diff
@@ -137,6 +138,22 @@ type Statistic
         Statistic.Kurtosis -> 4
         _ -> Nothing
 
+    ## PRIVATE
+    to_java self = case self of
+        Statistic.Count -> java_Statistic.Count
+        Statistic.Minimum -> java_Statistic.Minimum
+        Statistic.Maximum -> java_Statistic.Maximum
+        Statistic.Sum -> java_Statistic.Sum
+        Statistic.Mean -> java_Statistic.Mean
+        Statistic.Variance _ -> java_Statistic.Variance
+        Statistic.Standard_Deviation _ -> java_Statistic.StandardDeviation
+        Statistic.Skew _ -> java_Statistic.Skew
+        Statistic.Kurtosis -> java_Statistic.Kurtosis
+        Statistic.Covariance _ -> java_Statistic.Covariance
+        Statistic.Pearson _ -> java_Statistic.Pearson
+        Statistic.Spearman _ -> java_Statistic.Spearman
+        Statistic.R_Squared _ -> java_Statistic.R_Squared
```

Comment:
Any reason not to have the population flag on the Java enum and pass through.

Reply:
I didn't need it. :)

But I can do that

Reply:
In the next PR. It wasn't as straightforward as I hoped